webapp/src/components/Analysis/PerturbationTestingPreview.test.tsx

```diff
@@ -0,0 +1,243 @@
+import { fireEvent, screen, waitFor } from "@testing-library/react";
+import { renderWithRouterAndRedux } from "mocks/utils";
+import PerturbationTestingPreview from "./PerturbationTestingPreview";
+import { AvailableDatasetSplits } from "types/api";
+import { setupServer } from "msw/node";
+import {
+  getPertubationResponse,
+  getPertubationResponseWithnoTrainFailureRate,
+  getPertubationResponseWithFailureResponse,
+} from "mocks/api/mockPertubationAPI";
+import { formatRatioAsPercentageString } from "utils/format";
+
+const renderPerturbationTestingPreview = (
+  availableDatasetSplits: AvailableDatasetSplits
+) =>
+  renderWithRouterAndRedux(
+    <PerturbationTestingPreview
+      jobId="local"
+      pipeline={{ pipelineIndex: 0 }}
+      availableDatasetSplits={availableDatasetSplits}
+    />,
+    { route: "/local?pipeline_index=0", path: "/:jobId?:pipeline" }
+  );
+
+describe("Perturbation Testing Preview", () => {
```

Comment:
Detail, but when possible, I call the test the same as the function/component we are testing, so it's easy to search for and things like that.
```suggestion
describe("PerturbationTestingPreview", () => {
```

Reply:
make sense! will keep the consistent name as the component